### ISSUE TITLE
Password reset email and gmail auto-collapse

### ISSUE THREAD

**Issue by williamlief** [waldoCorp/easy-match#136]
The password reset email has identical text everytime it is sent, causing gmail to collapse and hide subsequent messages. Can we tweak the message to include the time when it was sent, and will the be enough to stop gmail's auto collapse behavior?

**Comment by bcerjan** [comment]
A cursory search indicates that gmail puts things in "conversations" when the subject line and sender are the same (or related to each other). This means we'd need to modify the subject line each time, which sort of isn't optimal. We could also (in theory) randomly change who is sending the email, but that feels super bad to do to. I feel like there should be some type of modification we can make to the email header to prevent this, but I haven't figured out what it would be yet.

**Comment by williamlief** [comment]
I did a little more reading and I think if we add the timestamp to the subject line it should stop this from happening.

**Comment by bcerjan** [comment]
Agreed, it might look a little funny but it should resolve the issue.

**Comment by bcerjan** [comment]
See 72a6764..b9cb45a. This should solve the issue but is a little ugly. Let me know what you think.
